commune_app/views.py

```diff
@@ -10,3 +10,7 @@ def chores(request):
     chores = Chore.objects.all()
     context = {'chores': chores}
     return render(request, 'commune_app/chores.html', context)
+
```

Comment:
since you replace the chores with a commune page
I think you need to delete lines # 9-12

Reply:
kept the context incase we use it inside the commune page